src-11ty/change-url.html

```diff
@@ -0,0 +1,33 @@
+---
+layout: base_card_with_hero
+javascript_source: my-change-url.js
+permalink: "redirect/_change/"
+description: Change the URL of your Home Assistant instance.
+---
+
+<style>
+  .card-content.placeholder {
+    min-height: 176px;
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+  }
+  .error {
+    color: #db4437;
+    font-weight: bold;
+  }
+</style>
+
+<h1 class="card-header">My Home Assistant</h1>
+
+<my-change-url>
+  <div class="card-content placeholder">
+    <p>
+      Configure My Home Assistant by entering the URL of your Home Assistant
+      instance.
+    </p>
+    <p>Loading...</p>
```

Comment:
This is what the placeholder renders:
![Screenshot 2021-03-05 at 20 53 30](https://user-images.githubusercontent.com/1444314/110195816-df7dfc00-7df4-11eb-83a3-ef3bf6d2e337.png)

This is what the loaded look is:
![Screenshot 2021-03-05 at 20 53 34](https://user-images.githubusercontent.com/1444314/110195829-ea389100-7df4-11eb-9520-2fc69adc5240.png)

They should match. We don't want the "Note" to be pushed down, so the placeholder for the input should be same height as it will be.

Reply:
We have 2 cases, I now did it based on the other one

Reply:
I'm confused. I thought that this url was only used for when you had no instance URL configured and you came to a redirect url ? 

Oh, this is for when you click the pencil icon? Makes sense 👍